modules/apps/style-book/style-book-service/src/main/java/com/liferay/style/book/service/impl/StyleBookEntryLocalServiceImpl.java

```diff
@@ -303,6 +309,55 @@ public StyleBookEntry updateDefaultStyleBookEntry(
 		return styleBookEntryPersistence.update(styleBookEntry);
 	}
 
+	@Override
+	public StyleBookEntry updateFrontendTokenDefinitionId(
+			long styleBookEntryId, String frontendTokenDefinitionId)
+		throws PortalException {
+
+		StyleBookEntry styleBookEntry =
+			styleBookEntryPersistence.findByPrimaryKey(styleBookEntryId);
+
+		styleBookEntry.setModifiedDate(new Date());
+		styleBookEntry.setFrontendTokenDefinitionId(frontendTokenDefinitionId);
+
+		StyleBookEntry draftStyleBookEntry = fetchDraft(styleBookEntry);
+
+		if (draftStyleBookEntry != null) {
+			draftStyleBookEntry.setModifiedDate(new Date());
+			draftStyleBookEntry.setFrontendTokenDefinitionId(
+				frontendTokenDefinitionId);
+
+			updateDraft(draftStyleBookEntry);
+		}
+
+		return styleBookEntryPersistence.update(styleBookEntry);
+	}
+
+	@Override
+	public StyleBookEntry updateFrontendTokenDefinitionName(
+			long styleBookEntryId, String frontendTokenDefinitionName)
+		throws PortalException {
+
+		StyleBookEntry styleBookEntry =
+			styleBookEntryPersistence.findByPrimaryKey(styleBookEntryId);
+
+		styleBookEntry.setModifiedDate(new Date());
+		styleBookEntry.setFrontendTokenDefinitionName(
+			frontendTokenDefinitionName);
+
+		StyleBookEntry draftStyleBookEntry = fetchDraft(styleBookEntry);
+
+		if (draftStyleBookEntry != null) {
+			draftStyleBookEntry.setModifiedDate(new Date());
+			draftStyleBookEntry.setFrontendTokenDefinitionName(
+				frontendTokenDefinitionName);
+
+			updateDraft(draftStyleBookEntry);
+		}
+
+		return styleBookEntryPersistence.update(styleBookEntry);
+	}
+
```

Comment:
Are these two operations requested in different moments? Otherwise, we could merge them.

Reply:
I think these methods are not necessary. there is no requirement to update the frontend token definition after a style book is created. 

Consider that a style book is created based on a theme CSS client extension. If the dev un-deploy the client extension, the style book will be unavailable until THE SAME client extension is deployed. 

Just remembering that we identify a client extension by its external reference code.

Reply:
I am not really sure, I just followed the pattern of the other update methods. Should I merge these or leave it as-is?

Reply:
Ah sorry I did not see your comment @thiago-buarqque, I will remove these methods

Reply:
I removed them in a new commit, let me know if it looks good. I also wanted to ask, is there a command to update tests in style-book-test? Please let me know too if I should send a new PR and combine some of these commits. @thiago-buarqque @algobob

Reply:
I updated some tests but its still causing issues in some places, I am not sure if the errors are related or not but will keep looking into this

Reply:
Also, do I need to add something similar to frontend-tokens-values.json test values for frontendTokenDefinitionId and frontendTokenDefinitionName for the tests in ExportImportStyleBookEntriesMVCResourceCommandTest or just pass blank or null values?

Reply:
I think we can just pass blank for now, since we're not focusing on Export/Import on this ticket